packages/api/src/clickhouse/index.ts

```diff
@@ -563,13 +563,13 @@ export const buildMetricsPropertyTypeMappingsModel = async (
 
 // TODO: move this to PropertyTypeMappingsModel
 export const doesLogsPropertyExist = (
-  property: string,
-  model: LogsPropertyTypeMappingsModel,
+  property?: string,
+  model?: LogsPropertyTypeMappingsModel,
```

Comment:
I wonder why these two args are optional

Reply:
probably better to add validation in route handler (routers/api/logs.ts:392)

![Screenshot 2023-11-14 at 10 28 41 PM](https://github.com/hyperdxio/hyperdx/assets/149748269/a62bf23c-6f5a-4731-b530-1388c9cef9f6)

Reply:
oh interesting, what if we set first arg of `doesLogsPropertyExist` type to `string | undefined` ?

Reply:
that should work